pkg/manifest/os.go

```diff
@@ -317,7 +319,7 @@ func (p *OS) getContainerSpecs() []container.Spec {
 }
 
 func (p *OS) serializeStart(packages []rpmmd.PackageSpec, containers []container.Spec, commits []ostree.CommitSpec) {
-	if len(p.packageSpecs) > 0 {
+	if len(p.getPackageSpecs()) > 0 {
```

Comment:
Why did this change?  Probably fine or even better this way, but curious.

Reply:
It changed because we were being inconsistent; some methods accessed `.packageSpecs`, some called `.getPackageSpecs()`.